File: src/main/java/pl/archala/testme/models/Answer.java
package pl.archala.testme.models;

import lombok.NoArgsConstructor;

import javax.persistence.Entity;

@Entity
public class Answer extends AbstractEntity<Long> {

    private String content;
    private boolean correctness = false;

    public Answer() {
    }

    public Answer(String content) {
        this.content = content;
    }

    public Answer(String content, boolean correctness) {
        this.content = content;
        this.correctness = correctness;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public boolean isCorrectness() {
        return correctness;
    }

    public void setCorrectness(boolean correctness) {
        this.correctness = correctness;
    }
}

File: app-angular/src/app/models/answer.ts
export class Answer {
  id: any;
  content!: string;
  correctness!: boolean;

  constructor(content: string, correctness: boolean) {
    this.content = content;
    this.correctness = correctness;
  }

  toString() {
    return "id=" + this.id + ", content=" + this.content + ", correctness=" + this.correctness;
  }
}
